src/main/java/codeu/model/data/User.java

```diff
@@ -23,36 +24,42 @@
   private final String name;
   private final String passwordHash;
   private final Instant creation;
+  private long messageCount;
+  private boolean isAdmin;
   private  String description;
 
-
   /**
    * Constructs a new User.
    *
-   * @param id the ID of this User
-   * @param name the username of this User
+   * @param id           the ID of this User
+   * @param name         the username of this User
    * @param passwordHash the password hash of this User
+   * @param creation     the creation time of this User
    * @param creation the creation time of this User
    * @param description of the username from profile section
    */
-  public User(UUID id, String name, String passwordHash, Instant creation, String description) {
```

Comment:
Lets make sure we are merging correctly. We don't want to cause any regressions. Here you are deleting the description param which is definitely what we do not want to do. I would say, start again with this class will the most recent master as the base and make your changes from there.